File: Examples/CSharp/ProgrammersGuide/Working-with-Pages/GetVisioPagebyName.cs
using Aspose.Diagram;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Aspose.Diagram.Examples.CSharp.ProgrammersGuide.Working_with_Pages
{
    public class GetVisioPagebyName
    {
        public static void Run()
        {
            // ExStart:GetVisioPagebyName
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_VisioPages();

            // Call the diagram constructor to load diagram from a VSDX file
            Diagram diagram = new Diagram(dataDir + "Drawing1.vsdx");

            // Set page name
            string pageName = "Flow 2";
            // Get page object by name
            Page page2 = diagram.Pages.GetPage(pageName);
            // ExEnd:GetVisioPagebyName
        }
    }
}

File: Examples/VisualBasic/Working-with-Pages/GetVisioPagebyName.vb
Imports Aspose.Diagram
Imports System
Public Class GetVisioPagebyName
    Public Shared Sub Run()
        ' ExStart:GetVisioPagebyName
        ' The path to the documents directory.
        Dim dataDir As String = RunExamples.GetDataDir_VisioPages()

        ' Call the diagram constructor to load diagram from a VDX file
        Dim diagram As New Diagram(dataDir & Convert.ToString("Drawing1.vsdx"))

        ' Set page name
        Dim pageName As String = "Flow 2"
        ' Get page object by name
        Dim page2 As Page = diagram.Pages.GetPage(pageName)
        ' ExEnd:GetVisioPagebyName
    End Sub
End Class
